packages/manager/modules/pci/src/projects/project/storages/databases/components/editable-text/editable-text.html

```diff
@@ -0,0 +1,23 @@
+<div class="d-flex align-items-center justify-content-between">
+    <oui-skeleton ng-if="$ctrl.pending"></oui-skeleton>
+    <span
+        class="w-100"
+        ng-if="!$ctrl.showInput && !$ctrl.pending"
+        data-ng-bind="$ctrl.inputValue"
+    ></span>
+    <span ng-if="$ctrl.showInput && !$ctrl.pending">
+        <input
+            id="{{$ctrl.id}}"
```

Comment:
**suggestion:** since it is a constant, one-time binding can be used

id="{{:: $ctrl.id}}"

Reply:
Done here: https://github.com/ovh/manager/pull/6399/commits/7413ebf2ff689776b8625102f24be891dffb6a3a